hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMAllocateBlockRequest.java

```diff
@@ -78,6 +80,29 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     Preconditions.checkNotNull(allocateBlockRequest);
 
     KeyArgs keyArgs = allocateBlockRequest.getKeyArgs();
+    String keyPath = keyArgs.getKeyName();
+    BucketLayout bucketLayout = getBucketLayout();
+    LOG.info("Bucket Layout: " + bucketLayout);
```

Comment:
Please change the priority to DEBUG and do like,

`LOG.debug("Bucket Layout: {}", bucketLayout);`

Reply:
Done